dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -93,15 +113,20 @@ object Datadog {
      * Initializes the Datadog SDK.
      * @param context your application context
      * @param clientToken your API key of type Client Token
-     * @param endpointUrl (optional) the endpoint url to target, or null to use the default. Possible values are
-     * [DATADOG_US_LOGS], [DATADOG_EU_LOGS] or a custom endpoint.
+     * @param logsEndpointUrl (optional) the logs endpoint url to target,
+     * or null to use the default.
+     * Possible values are [DATADOG_US_LOGS], [DATADOG_EU_LOGS] or a custom endpoint.
+     * @param tracesEndpointUrl (optional) the traces endpoint url to target,
+     * or null to use the default.
+     * Possible values are [DATADOG_US_TRACES], [DATADOG_EU_TRACES] or a custom endpoint.
      */
     @JvmStatic
     @JvmOverloads
     fun initialize(
         context: Context,
         clientToken: String,
-        endpointUrl: String? = null
+        logsEndpointUrl: String? = null,
```

Comment:
This method will be changed in a future task. Probably after we have a stable tracer. As discussed we will no longer provide all these parameters in the `initialize` method but instead we will provide an SdkConfig based on a Builder pattern.